src/app/pages/home/home.js

```diff
@@ -11,7 +12,24 @@ import {template} from './home.hbs';
 export default class Home extends BaseView {
 
     onRender() {
-        // this.regions.featured.show(new CarouselView());
+        this.updateHeaderStyle();
+        window.addEventListener('scroll', this.updateHeaderStyle.bind(this));
+    }
+
+    updateHeaderStyle() {
+        if (window.pageYOffset > 50 && !appView.header.isPinned()) {
```

Comment:
The secondary nav height is now dynamically calculated.  `10` is just the conversion from `px` to `rem` which is defined for this project.